monty/exts/info/meta.py

```diff
@@ -109,7 +109,15 @@ async def invite(
         raw_link: bool = False,
         ephemeral: bool = True,
     ) -> None:
-        """Generate an invite link to invite monty."""
+        """Generate an invite link to invite monty.
```

Comment:
if we're doing this, should be ``Monty``

Reply:
my bad, one second